src/video.tsx

```diff
@@ -90,13 +168,28 @@ const Video = React.forwardRef<HTMLVideoElement, VideoProps>(
                 playsInline
                 disablePictureInPicture
                 disableRemotePlayback
-                autoPlay={autoPlay}
+                autoPlay={autoPlay === 'streaming'}
```

Comment:
does this still make sense? you are already calling play manually in `onLoadFinish` function

Reply:
Good call. It's old code